runner/metadata_test.go

```diff
@@ -252,6 +252,91 @@ func TestMetadata_Set(t *testing.T) {
 	}
 }
 
+func TestMetadata_Set_interpreter(t *testing.T) {
+	tests := []struct {
+		name    string
+		config  string
+		env     string
+		want    []string
+		wantErr string
+	}{
+		{
+			name: "defaults",
+		},
+		{
+			name:   "config file",
+			config: `interpreter: bash`,
+			want:   []string{"bash"},
+		},
+		{
+			name: "env",
+			env:  `/usr/bin/env zsh`,
+			want: []string{"/usr/bin/env", "zsh"},
+		},
+		{
+			name:   "config file and env",
+			config: `bash`,
+			env:    `/usr/bin/env zsh`,
+			want:   []string{"/usr/bin/env", "zsh"},
```

Comment:
Is this a typo? (`config: bash`), want `zsh`...

Reply:
It was meant to test environment variable precedence, but this test case is now removed.